src/OpenTelemetry.SemanticConventions/SemanticConventionsAttributes.cs

```diff
@@ -0,0 +1,5448 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// <auto-generated>This file has been auto generated from scripts/templates/SemanticConventionsAttributes.cs.j2</auto-generated>
+
+#pragma warning disable CS1570 // XML comment has badly formed XML
+
+namespace OpenTelemetry.SemanticConventions
```

Comment:
Some time ago I have seen discussion to include semantic convention version in namespace to avoid breaking changes. Newer versions should just incrementally add new files.

Reply:
IMO a lot of work needs to be done before this is really useful 🤣 But I would suggest we separate this effort to move it here to contrib (with minor updates) and improving it further (can be done in follow-up PRs)?

Reply:
Sure, but I would wait with re-relasing package until we have something at least usable.

Reply:
Agree with you on that 👍